components/nodemanager-service/tests/project_filtering_test.go

```diff
@@ -375,11 +375,20 @@ func TestListProjectFilteringIngestedNodes(t *testing.T) {
 
 				// Get all the node IDs returned.
 				actualNodeIDs := []string{}
+				actualManuallNodeIDs := []string{}
 				for _, node := range nodesResponse.Nodes {
-					actualNodeIDs = append(actualNodeIDs, node.Id)
+					if node.Manager == "" {
+						actualNodeIDs = append(actualNodeIDs, node.Id)
+					} else {
+						actualManuallNodeIDs = append(actualManuallNodeIDs, node.Id)
+					}
+				}
+
+				for _, nodeID := range manualNodeIds {
+					assert.Contains(t, actualManuallNodeIDs, nodeID)
 				}
```

Comment:
Should `manualNodeIds` and `actualManualNodeIDs` contain the same elements? Should this be a `ElementsMatch` assertion like below?

Reply:
there could be more nodes in the actualManualNodeIDs than just the ones added in this test -- from other tests that ran first/scans finishing